flink-libraries/flink-state-processing-api/src/test/java/org/apache/flink/state/api/output/SnapshotUtilsTest.java

```diff
@@ -45,18 +49,51 @@
     private static final List<String> EXPECTED_CALL_OPERATOR_SNAPSHOT =
             Arrays.asList("prepareSnapshotPreBarrier", "snapshotState", "notifyCheckpointComplete");
 
+    private static final List<SnapshotType> EXPECTED_SNAPSHOT_TYPE = new ArrayList<>();
+
     @Rule public TemporaryFolder folder = new TemporaryFolder();
 
     private static final List<String> ACTUAL_ORDER_TRACKING =
             Collections.synchronizedList(new ArrayList<>(EXPECTED_CALL_OPERATOR_SNAPSHOT.size()));
 
+    private static final List<SnapshotType> ACTUAL_SNAPSHOT_TYPE = new ArrayList<>();
```

Comment:
I don't think we need a list to hold the snapshot type. Moreover, the expected snapshot type is what we passed in, an no need to introduce another list of `EXPECTED_SNAPSHOT_TYPE`.